homeassistant/components/suez_water/strings.json

```diff
@@ -6,7 +6,13 @@
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]",
           "counter_id": "Counter id"
-        }
+        },
+        "data_description": {
+          "username": "Enter your login associated with your Tout sur mon eau account",
+          "password": "Enter your password associated with your Tout sur mon eau account",
+          "counter_id": "Enter your counter id (ex: 12345678). Should be found automatically during setup, if not see integration documentation for more information"
```

Comment:
```suggestion
          "counter_id": "Enter your meter id (ex: 12345678). Should be found automatically during setup, if not see integration documentation for more information"
```

Reply:
I'll have to do it everywhere else code and docs, but if it's ok with you I'll keep that in another PR